phpBB/phpbb/user.php

```diff
@@ -369,7 +369,24 @@ function setup($lang_set = false, $style_id = false)
 		}
 
 		// Is board disabled and user not an admin or moderator?
-		if ($config['board_disable'] && !defined('IN_INSTALL') && !defined('IN_LOGIN') && !defined('SKIP_CHECK_DISABLED') && !$auth->acl_gets('a_', 'm_') && !$auth->acl_getf_global('m_'))
+		// Check acp setting who has access: only admins "case: 0", plus global moderators "case: 1" and plus moderators "case: 2"
+		$board_disable_access = (int) $config['board_disable_access'];
+
+		switch ($board_disable_access)
+		{
+			case 0:
+				$access_disabled_board = $auth->acl_gets('a_');
+				break;
```

Comment:
`break;` should be aligned like case and there should be an empty line after the break